src/app.ts

```diff
@@ -1,6 +1,8 @@
 import { autoinject } from "aurelia-framework";
 import { RouterConfiguration, Router } from "aurelia-router";
+import {PLATFORM} from 'aurelia-pal';
 import { I18N } from 'aurelia-i18n';
+import 'app.scss';
```

Comment:
this was needed, as the require in the html file was not working. see webpack config